core/pkg/sync/isync.go

```diff
@@ -47,4 +47,15 @@ type SourceConfig struct {
 	Selector    string `json:"selector,omitempty"`
 	Interval    uint32 `json:"interval,omitempty"`
 	MaxMsgSize  int    `json:"maxMsgSize,omitempty"`
+
+	OAuthConfig *OAuthCredentialHandler `json:"oauthConfig,omitempty"`
```

Comment:
```suggestion
	OAuthConfig *OAuthCredentialHandler `json:"oauth,omitempty"`
```

[nit] Naming it `oauthConfig` is somewhat redundant, as this is a configuration anyways, i thing we can simply call it `oauth`

// pair-review: @alexandraoberaigner @guidobrei

Reply:
agree